test.go

```diff
@@ -203,6 +207,18 @@ func (t *Test) SaveLatestCSV() {
 	}
 }
 
+// StoreLatestToPrometheus stores latest version results to prometheus pushgateway
+func (t *Test) StoreLatestToPrometheus() {
+	version := t.config.Versions[len(t.config.Versions)-1]
+	cli := NewPromClient(t.promConfig)
+	for _, q := range t.queries {
+		res := average(t.results[version][q.ID])
+		if err := cli.Dump(res, version, q.ID, t.ciConfig.Branch, t.ciConfig.Commit); err != nil {
+			panic(err)
```

Comment:
I would return an error instead of panicking here. Then if needed panic in the main

Reply:
Done